bash/scripts/get_latest_tag.sh

```diff
@@ -11,5 +13,12 @@ get-latest-tag() {
   else
     exit 1
   fi
+
+  # If tag already released, bail out
+  if curl -sf "https://versions.deis.com/v3/versions/stable/deis-${component}/${tag}"; then
```

Comment:
The only thing I'm concerned about is that this will exit 1 if versions.deis.com is down or unresponsive with no indication that it failed because it was down. Is there a way we can report the error that is returned here so we can determine next steps to remediate the issue?